### PR TITLE
Add version badges

### PR DESCRIPTION
**Pull request by DavidDudson**
There's currently no easy way to find out the latest versions. Hopefully these badges help.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 [![Build Status](https://platform-ci.scala-lang.org/api/badges/scalameta/scalameta/status.svg)](https://platform-ci.scala-lang.org/scalameta/scalameta)
 [![Join the chat at https://gitter.im/scalameta/scalameta](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/scalameta/scalameta?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+## Versions
+Release: ![Release](https://img.shields.io/maven-central/v/org.scalameta/scalameta_2.12.svg)
```
Comment (olafurpg) [inline README.md]: How about?

> See http://scalameta.org/#Gettingstarted for the latest stable and pre-release versions.

Those version numbers are copy-pasteable. The pre-release version is updated on every merge into master.

Comment (DavidDudson) [inline README.md]: Oh, I have not looked at the website for ages. I did not realise that was autogenerated. Ill close this PR.

Comment (olafurpg) [inline README.md]: I still think it's a good idea to link it from the readme, because I suspect that's where many people will look for the version number.

Comment (DavidDudson) [inline README.md]: @olafurpg It also confuses me that the meta page does not have paradise versions, you have to go to the tutorial site for that IIRC.

Comment (olafurpg) [inline README.md]: Same here, has confused me too. With #816, it should be possible to have a matrix with all versions in one place.

**Pull request closed without merging**